lib/absinthe/schema/notation/sdl_render.ex

```diff
@@ -5,13 +5,11 @@ defmodule Absinthe.Schema.Notation.SDL.Render do
 
   alias Absinthe.Blueprint
 
-  @line_width 120
-
-  def inspect(term, %{pretty: true}) do
+  def inspect(term, %{pretty: true, width: width}) do
     term
     |> render()
     |> concat(line())
-    |> format(@line_width)
+    |> format(width)
```

Comment:
Should this not be applied in the `Absinthe.Language.Render` module? You add the width to the Absinthe.Formatter but this is never passed on to the `SDL.Render` module.

There are two intermedate representations in Absinthe of schema/executable documents. The Language structs, which is the result of the lexer/parser. And the Blueprint structs, which is used to build schema's and execute them. 

Both representations can be rendered to the graphql string representation. The Language representation can be converted to executable documents. 
Blueprints can be converted back to the schema definition language.

Reply:
Ah, I overlooked that. I'm happy to make the change wherever it makes the most sense to you 😄 .

Reply:
https://github.com/absinthe-graphql/absinthe/blob/102360eb4dbc289f223cf1cbc64fc01f148721f8/lib/absinthe/language/render.ex#L12 this is probably what you're looking for.